tests/tests/test_monitor_client.py

```diff
@@ -657,3 +663,73 @@ def test_dbus_subsystem(self, monitor_commercial_setup_no_client):
             + devid
         )
         logger.info("test_dbus_subsystem: got CRITICAL alert email.")
+
+    def test_dbus_pattern_match(self, monitor_commercial_setup_no_client):
+        """Test the dbus subsystem"""
+        mailbox_path = "/var/spool/mail/local"
+        wait_for_alert_interval_s = 8
+        expected_from = "alert@mender.io"
+        dbus_name = "test"
+        user_name = "bugs.bunny@acme.org"
+        devid, _, _, mender_device = self.prepare_env(
+            monitor_commercial_setup_no_client, user_name
+        )
+        logger.info("test_dbus_subsystem: env ready.")
+
+        logger.info(
+            "test_dbus_subsystem: email alert on dbus signal pattern match scenario."
+        )
+        prepare_dbus_monitoring(mender_device, dbus_name, log_pattern="mender")
```

Comment:
another nit: technically it is not "log_pattern" but `dbus_pattern="mender"` right?

Reply:
Right, good catch. 
Thx